gsy_framework/solar_api_clients/solar_api_client_active.py

```diff
@@ -23,14 +23,16 @@ class SolarAPIClientActiveException(Exception):
 class SolarAPIClientActive(SolarAPIClientBase):
     """ETL for energy data from active solar API."""
 
-    @staticmethod
-    def _get_corresponding_historical_time_stamp(input_datetime: DateTime) -> DateTime:
+    def _set_use_historical_data(self, input_datetime: DateTime):
+        self._use_historical_data = ((input_datetime.year >= today().year) or
```

Comment:
Via the UI, the user can select future dates. I know, it does not make sense (we might think of disabling this because you are 100% correct, it does not make sense to do so). Should I create a FE ticket?

Reply:
If you can take the time to file a FE ticket it would be great! Thanks in advance!

Reply:
Done https://gridsingularity.atlassian.net/browse/FE-481